tests/nodeos_forked_chain_test.py

```diff
@@ -290,7 +290,7 @@ def getMinHeadAndLib(prodNodes):
 
         if producerToSlot[lastBlockProducer]["count"]!=inRowCountPerProducer:
             Utils.errorExit("Producer %s, in slot %d, expected to produce %d blocks but produced %d blocks.  At block number %d." %
-                            (blockProducer, slot, inRowCountPerProducer, producerToSlot[lastBlockProducer]["count"], blockNum))
+                            (lastBlockProducer, slot, inRowCountPerProducer, producerToSlot[lastBlockProducer]["count"], blockNum-1))
 
```

Comment:
Add to the PR description that you also updated error message of forked_chain_test.